Backport minor regression in 4.1.28 powerpc kernel base.

See discussion on the powerpc kernel mailing list:
https://lists.ozlabs.org/pipermail/linuxppc-dev/2016-July/145998.html

This commit causes a regression:

  commit f49eb503f0f9ab66874a81992b14249a1c59b6ad
  Author: Anton Blanchard <anton@samba.org>
  Date:   Fri Apr 15 12:08:19 2016 +1000

    powerpc: Update TM user feature bits in scan_features()

This patch can likely be dropped for the next stable 4.1.y release.

diff --git a/arch/powerpc/kernel/prom.c b/arch/powerpc/kernel/prom.c
index 28dbbb0..9b19d96 100644
--- a/arch/powerpc/kernel/prom.c
+++ b/arch/powerpc/kernel/prom.c
@@ -166,8 +166,7 @@ static struct ibm_pa_feature {
 	 * we don't want to turn on TM here, so we use the *_COMP versions
 	 * which are 0 if the kernel doesn't support TM.
 	 */
-	{CPU_FTR_TM_COMP, 0, 0,
-	 PPC_FEATURE2_HTM_COMP|PPC_FEATURE2_HTM_NOSC_COMP, 22, 0, 0},
+	{CPU_FTR_TM_COMP, 0, 0, PPC_FEATURE2_HTM_COMP, 22, 0, 0},
 };
 
 static void __init scan_features(unsigned long node, const unsigned char *ftrs,
